v4l2/v4l2_wrapper.h

```diff
@@ -27,6 +27,7 @@
 #else
 #include <EGL/egl.h>
 #endif
+#include "v4l2codec_device_ops.h"
```

Comment:
my mistake, removed it in v4l2_wrapper.h.
only include it in v4l2_wrapper.cpp is necessary